src/wrappers.cpp

```diff
@@ -93,3 +94,40 @@ Eigen::VectorXd qkde1d_cpp(const Eigen::VectorXd& x,
 
     return q;
 }
+
+// Bandwidth for Kernel Density Estimation
+//' @param x vector of observations
+//' @param grid_size number of equally-spaced points over which binning is
+//' performed to obtain kernel functional approximation
+//' @return the selected bandwidth
+//' @noRd
+// [[Rcpp::export]]
```

Comment:
Not sure this qualifies as wrapper ;)

Reply:
Yeah, I wondered after looking at `qkde1d_cpp` what would qualify as a wrapper.

Initially, I add a `dpik` function in `dpik.hpp` and then a `dpik_cpp` function that simply called `dpik`, but it felt really useless.

Up to you.

Reply:
OK, I see your point. The actual wrapper would be `select_bw()`. How about we port this to to cpp as `select_bw_cpp()` and move `dpik` to `dpik.hpp`?